src/install-elixir.ps1

```diff
@@ -0,0 +1,5 @@
+Write-Output "Installer for Elixir for Windows not available"
```

Comment:
He/she/it who comes after is more than welcome to take a stab at this. For the time being I preferred to be explicit in saying Elixir is not available.

Reply:
It's also not clear whether the elixir web installer can do unattended installs to boot.

Reply:
I thought the main selling point of Elixir web installer was that it would install OTP for you as well. Which is not what we want as we want more control into particular OTP versions.

I think we could treat Elixir tarball similar to Erlang tarballs, we unpack it somewhere and add bin/ to path. Elixir ships with Windows scripts (https://github.com/elixir-lang/elixir/tree/master/bin). I don't know if it is relevant but it looks like it only has .ps1 for mix and not others.

THat being said, probably out of the scope of this PR.

Reply:
Ahh yeah, that makes sense. I've never installed elixir on windows, but yeah I would think the precompiled tarball should "just work" ™️

Reply:
> THat being said, probably out of the scope of this PR.

If it very simple to add (ie.I have instructions or pointers) I don't mind doing it - in that sense it's not out-of-scope because Elixir was part of the mix already, but I don't have an easy way to do it, since I might miss something. With Erlang/OTP I knew exactly what I was looking for.

Reply:
That said, can any of you give me pointers as to how this could be achieved?

Reply:
This seems to do the trick!

https://github.com/wojtekmach/test_erlang/blob/aa81bbbc7654af98adba2583061179ffd1b60624/.github/workflows/main.yml#L25:L40

https://github.com/wojtekmach/test_erlang/runs/2894176893?check_suite_focus=true

Reply:
we should of course download from repo.hex.pm, same as we do for Linux, because we want to grab Elixir compiled against a matching OTP version.

in fact, do we even need `install-elixir.ps1`? perhaps it's easier to do everything in JS because it is supposed to work cross-platform, all we need to do is unzip and add the bin/ directory to $GITHUB_PATH. Again I'm not well versed to comment on the implementation, pick whatever is the simplest and most maintainable.

Reply:
Yeah, the other day I was wondering why some stuff was written as Shell script vs JavaScript, myself. I think it should only be so when execution is needed in the target platform (e.g. I want to get the Elixir version from Elixir to make sure the installation process was Ok) - but most other stuff could be converted to JS... anyway, I've linked this to the newer issue, so your comments will be _ported_ over also. Thanks for the help.